### ISSUE TITLE
rabbitmq_vhost.present state broken in v2015.5.2

### ISSUE THREAD

**Issue by copelco** [caktus/django-project-template#182]
AFAICT, ``rabbitmq_vhost.present`` always throws a ``TypeError`` due to not passing an interable into [this any() function](https://github.com/saltstack/salt/blob/v2015.5.2/salt/states/rabbitmq_vhost.py#L84):

```
out:           ID: broker-vhost
out:     Function: rabbitmq_vhost.present
out:         Name: traffic_stops_staging
out:       Result: False
out:      Comment: An exception occurred in this state: Traceback (most recent call last):
out:                 File "/usr/lib/python2.7/dist-packages/salt/state.py", line 1563, in call
out:                   **cdata['kwargs'])
out:                 File "/usr/lib/python2.7/dist-packages/salt/states/rabbitmq_vhost.py", line 84, in present
out:                   if any(user, owner, conf, write, read):
out:               TypeError: any() takes exactly one argument (5 given)
out:      Started: 15:11:11.376117
out:     Duration: 1.642 ms
out:      Changes:   
```

This has been fixed on [develop](https://github.com/saltstack/salt/commit/01c99ad7677991cee541f2e9da7a0751674d0fbb#diff-b90da430367e42d08723b352c9128b8e).

**Comment by mlavin** [comment]
Only options seem to be use 2015.5.0 or wait until 2015.5.3 is released. Maybe we should update the default in the fabfile.

**Comment by mlavin** [comment]
Another potential problem with rabbitmq_user https://github.com/saltstack/salt/issues/25125

**Comment by marek-obuchowicz** [comment]
The problem with not detecting existing users (https://github.com/saltstack/salt/issues/25125) also appears to affect rabbimq_vhost - in the example below, salt tries to create vhost, even if it exists, producing an error:
[INFO    ] Executing state rabbitmq_vhost.present for /DE_testing_zed
stdin: is not a tty
[INFO    ] Executing command 'rabbitmqctl list_vhosts' as user 'root' in directory '/root'
[DEBUG   ] output: Listing vhosts ...
/
/DE_development_zed
/DE_testing_zed
stdin: is not a tty
[INFO    ] Executing command 'rabbitmqctl add_vhost /DE_testing_zed' as user 'root' in directory '/root'
[ERROR   ] Command 'rabbitmqctl add_vhost /DE_testing_zed' failed with return code: 2
[ERROR   ] output: Creating vhost "/DE_testing_zed" ...
Error: vhost_already_exists: /DE_testing_zed

**Comment by mlavin** [comment]
Confirmed this is still broken in 2015.5.3.

**Comment by dpoirier** [comment]
Keep your fingers crossed, but this appears to be fixed in Salt v2015.8.0rc2; I just tried it.

**Comment by mlavin** [comment]
It looks like this fix was backported and included in the 2015.5.5 release http://docs.saltstack.com/en/latest/topics/releases/2015.5.5.html
